apps/networkmonitor/networkmonitor_config.nim

```diff
@@ -32,12 +32,46 @@ type
       defaultValue: ""
       name: "dns-discovery-url" }: string
 
+    pubsubTopics* {.
+      desc: "Default pubsub topic to subscribe to. Argument may be repeated."
+      name: "pubsub-topic" .}: seq[string]
+
     refreshInterval* {.
       desc: "How often new peers are discovered and connected to (in seconds)",
       defaultValue: 5,
       name: "refresh-interval",
       abbr: "r" }: int
 
+    rlnRelay* {.
+        desc: "Enable spam protection through rln-relay: true|false",
+        defaultValue: true
+        name: "rln-relay" }: bool
+
+    rlnRelayCredIndex* {.
```

Comment:
since network monitor will just monitor and not publish messages, maybe `rlnRelayCredIndex` can be removed?

Reply:
Yeah, makes sense, removed